### ISSUE TITLE
Microsoft Graph especially OneDrive

### ISSUE THREAD

**Issue by farukaydin** [automatisch/automatisch#913]
### Discussed in https://github.com/automatisch/automatisch/discussions/820

<div type='discussions-op-text'>

<sup>Originally posted by **orgildinio** December 27, 2022</sup>
Hello.
Microsoft sharepoint, Onedrive integration would be nice.
Its widely used in companies with o365 subscription. If integration is good, Automatisch would be Power Automation from Microsoft its own game</div>

Existing upvote count from GitHub discussions: 1

**Comment by sid351** [comment]
I agree MS Graph integration would be useful, but only to the limit that the MS Graph API allows - this wouldn't be the cure-all of automations for Microsoft 365.  With smart use of HTTP Requests and handling Webhooks it could link in with Power Automate very nicely though.
